packages/core/src/sizePreferences/redux/actionTypes.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @module listingFacets/actionTypes
+ * @category Listing Facets
+ * @subcategory Action types
+ */
+
+/** Action type dispatched when the get listing facets request fails. */
```

Comment:
I think could be added RESET Action to reset this redux.
Check sizeguides.


```suggestion
/** Action type dispatched when the reset listing facets occurs. */
export const RESET_LISTING_FACETS = '@farfetch/blackout-core/RESET_LISTING_FACETS';
```

Reply:
Done! 👍🏼